### ISSUE TITLE
Make use of the new Krist API

### ISSUE THREAD

**Issue by xAnavrins** [apemanzilla/KWallet#6]
The format returned by ?listtx has been changed, which breaks the transactions page.
So might as well make use of the new API, though since there's a cap for how many transaction is returned, the graph window won't work.
http://krist.ceriat.net/docs/

**Comment by dmarcuse** [comment]
Once I get some time (and motivation) I'm just going to rewrite this whole project, and probably KristAPI as well.

**Comment by LoganDark** [comment]
You should work on a new interface, such as a modern one from krist.ceriat.net. I made a mockup of what I mean, shown here:

!["Quick" mockup](https://cloud.githubusercontent.com/assets/4723091/15988758/cf754d18-3010-11e6-8cf3-97f0bda4f832.png)

**Comment by dmarcuse** [comment]
Yeah, I have a list of things to do. Right now I'm a bit busy with school though.

**Comment by LoganDark** [comment]
This is a mockup of my wallet in the new UI (with a bit of tweaks)

![Another mockup](https://cloud.githubusercontent.com/assets/4723091/15988837/f0c181fa-3013-11e6-9df0-977901356ce0.png)

**Comment by dmarcuse** [comment]
Yeah, I saw that the first time :stuck_out_tongue: 

Maybe I'll try out JavaFX.

**Comment by LoganDark** [comment]
I could try and create the front-end with HTML & CSS. You'd have to script it though.

**Comment by dmarcuse** [comment]
I'd like to keep KWallet in regular old Java for now. You could see if you could write the web wallet for the official Krist node though - http://github.com/lemmmy/krist

**Comment by LoganDark** [comment]
yeah, but I can't code interactive web pages, only static :/

**Comment by dmarcuse** [comment]
I'm sure if you offer to do the templates and CSS there would be someone to do the scripting for it.
